Source/Rio.Web/src/app/pages/parcel-ledger-create-from-spreadsheet/parcel-ledger-create-from-spreadsheet.component.ts

```diff
@@ -75,22 +74,33 @@ export class ParcelLedgerCreateFromSpreadsheetComponent implements OnInit {
     this.document.getElementById("CSV-upload").click();
   }
 
-  private clearErrorAlerts() {
-    if (!this.alertsCountOnLoad) {
-      this.alertsCountOnLoad = this.alertService.getAlerts().length;
+  private validateEffectiveDateAndWaterTypeID() {
+    var formValid = true;
+    if (!this.effectiveDate) {
```

Comment:
Why isn't this handled by the server side validators?

Reply:
Validation moved to API